src/app/_components/fixedlayout.tsx

```
@@ -26,22 +28,28 @@ export default function FixedLayout({
+    state === "default"
+      ? " pt-21.75"
+      : state === "detail"
+        ? "pt-15.5"
+        : "py-auto";
   return (
     <div
-      className={cn(
-        "flex min-h-screen flex-col pt-21.75 text-white",
-        !disablePadding && "px-5",
-      )}
+      className={cn(`${paddingStyle} flex min-h-screen flex-col px-5 text-white`, !disablePadding && "px-5")}
```

ESLint

[error] Replace ``${paddingStyle}·flex·min-h-screen·flex-col·px-5·text-white`,·!disablePadding·&&·"px-5"` with `⏎········`${paddingStyle}·flex·min-h-screen·flex-col·px-5·text-white`,⏎········!disablePadding·&&·"px-5",⏎······`

(prettier/prettier)